Vagrantfile

```diff
@@ -166,28 +166,14 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.define "debian8-agent", autostart: false do |agent|
     agent.vm.box = "debian/jessie64"
+    agent.vm.box_version = '8.11.0'
```

Comment:
Add a note and a link to this PR that 8.11.1 is broken, so we know why it is pinned and perhaps can unpin when 8.12 comes out or whenever the fix is ready.

Reply:
Comment added.